cmd/sonobuoy/app/imageversion.go

```diff
@@ -79,7 +80,10 @@ func (c *ConformanceImageVersion) Get(client discovery.ServerVersionInterface) (
 			return "", err
 		}
 
-		return version.GitVersion, nil
+		// NOTE: Until the kube-conformance container is pushed upstream we can't
+		// guarantee alignment with exact versioning see https://github.com/heptio/kube-conformance/issues/25
+		// for more details
+		return fmt.Sprintf("v%s.%s", version.Major, version.Minor), nil
 	}
 	return string(*c), nil
 }
```

Comment:
Do we want to constrain validateVersion to minor releases too? Or is that fine as is

Reply:
It was fine for validation so I didn't change it.  Ideally I'd like to move the code back to your original PR in the future if we can push upstream to publish the testing container per release.